### ISSUE TITLE
Use Case #2 happy path - variables selection

### ISSUE THREAD

**Issue by HelenClifton** [CCI-Tools/cate#591]
### Expected behavior

*Describe what you expected Cate to do.*
Selected **tseries_point** operation on Cloud dataset and selected variable **cfc**. Added the step.
Selected **tseries_point** operation on soil moisture dataset. In "New Operation Step" dialog, expected variable selection box to be empty

### Actual behavior

Variable selection contains the cfc variable from the Cloud dataset (which does not exist in soil moisture dataset)

![capture_uc02](https://user-images.githubusercontent.com/29569434/37902766-fd2f3058-30e4-11e8-9615-517a5f488fb6.PNG)

### Steps to reproduce the problem

Attached here:
[steps to reproduce.docx](https://github.com/CCI-Tools/cate/files/1847525/steps.to.reproduce.docx)

### Specifications 

cate-2.0.0.dev.7
Windows 7 Professional

**Comment by forman** [comment]
Yes. If variables can not be found in the selected dataset, fields of type `VarNamesLike` or `VarName` should be cleared out.

**Comment by bwuerzler** [comment]
Hello. I also experienced this error, but in a slightly different way. After the visualization of some cfc data i removed it from workspace, opened and tried to visualize some aerosol data. However, the created plots showed the cfc data displayed before, no matter how often i retried to visualize the aerosol data. For me it seems like the cfc data was not completely removed from the session and in the end i had to restart cate.
